docs/2_configuration.md

```diff
@@ -90,19 +90,20 @@ Need to make some changes to how Pay is used? You can create an initializer `con
 
 ```ruby
 Pay.setup do |config|
-  # For use in the receipt/refund/renewal mailers
-  config.business_name = "Business Name"
-  config.business_address = "1600 Pennsylvania Avenue NW"
-  config.application_name = "My App"
-  config.support_email = "helpme@example.com"
-
   config.send_emails = true
+  config.generate_receipts = true
```

Comment:
Is there a use case where you'd have the `receipts` gem installed and _not_ want to include receipts in your emails?